docs/KVSet/README.md

```diff
@@ -0,0 +1,6 @@
+# Azure App Configuration 'appconfig/kvset' profile
+Azure App Configuration can be exported and imported through azure CLI using the `appconfig/kvset` profile.
+The following is the JSON schema used to import and export using `appconfig/kvset` profile - [KVSet.v1.0.0.schema.json](./KVSet.v1.0.0.schema.json).
```

Comment:
It's okay to mention Azure CLI for now. In the future, we will support it in other ways as well.

Just to make sure it's clear, we probably should mention this profile is for importing from files or exporting to files. (It doesn't apply in other import/export scenarios.)

Reply:
I added a note below this to mention it is only available when importing/exporting to a JSON file.